website/addons/base/views.py

```diff
@@ -643,6 +643,10 @@ def addon_view_or_download_file(auth, path, provider, **kwargs):
     if action == 'download':
         return redirect(file_node.generate_waterbutler_url(**dict(extras, direct=None, version=version.identifier)))
 
+    if action == 'get_guid':
```

Comment:
See https://openscience.atlassian.net/browse/OSF-6411?focusedCommentId=35797&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-35797 : Can we just create guids for the selected files and check those out?